VXLAN: allow multiple remotes (#12052)
canonical/lxd: doc/reference/network_bridge.md

`tunnel.NAME.local`                  | string    | `gre` or `vxlan`      | -                         | Local address for the tunnel (not necessary for multicast `vxlan`)
`tunnel.NAME.port`                   | integer   | `vxlan`               | `0`                       | Specific port to use for the `vxlan` tunnel
`tunnel.NAME.protocol`               | string    | standard mode         | -                         | Tunneling protocol: `vxlan` or `gre`

> `tunnel.NAME.remote`                 | string    | `gre` or `vxlan`      | -                         | Remote address for the tunnel. For a `vxlan` tunnel it is possible to specify multiple IP addresses separated by comma (not necessary for multicast `vxlan`)

Reviewer: ```suggestion
`tunnel.NAME.remote`                 | string    | `gre` or `vxlan`      | -                         | Remote address for the tunnel - for a `vxlan` tunnel, you can specify multiple IP addresses separated by comma (not necessary for multicast `vxlan`)
```
